packages/react/globals.css

```diff
@@ -15,5 +15,9 @@
 
     --muted: 210 40% 96.1%;
     --muted-foreground: 215.4 16.3% 46.9%;
+
+    --accent-foreground: 0 0% 9%;
+    --destructive: 0 84.2% 60.2%;
+    --destructive-foreground: 0 0% 98%;
```

Comment:
This is only used for tests right? If so, can we move it to the spec dir?

Reply:
ping on this one @jadewale -- this file isn't required to be imported by end users right, just used for our tests?

Reply:
I think end users will need this as part of the tailwind set up. The tailwind config refers to these colors in your global.css